beetsplug/mpdstats.py

```diff
@@ -263,25 +263,27 @@ def on_play(self, status):
         played, duration = map(int, status['time'].split(':', 1))
         remaining = duration - played
 
-        if self.now_playing and self.now_playing['path'] != path:
-            skipped = self.handle_song_change(self.now_playing)
-            # mpd responds twice on a natural new song start
-            going_to_happen_twice = not skipped
-        else:
-            going_to_happen_twice = False
-
-        if not going_to_happen_twice:
-            self._log.info(u'playing {0}', displayable_path(path))
-
-            self.now_playing = {
-                'started':    time.time(),
-                'remaining':  remaining,
-                'path':       path,
-                'beets_item': self.get_item(path),
-            }
-
-            self.update_item(self.now_playing['beets_item'],
-                             'last_played', value=int(time.time()))
+        if self.now_playing:
+            if self.now_playing['path'] != path:
+                self.handle_song_change(self.now_playing)
+            else:  # in case we got mpd play event with same song playing multiple times
+                # assume low diff means redundant second play event after natural song start
```

Comment:
I know this is a tiny thing, but please use full sentences ("Starting with a capital letter, and ending with a period.") in comments.